docs/semconv/database.md

```diff
@@ -302,13 +310,92 @@ This metric is [optional](https://github.com/open-telemetry/semantic-conventions
 | Attribute            | Type   | Description                 | Example                | Requirement Level |
 |----------------------|--------|-----------------------------|------------------------|-------------------|
 | `tablespace_name`    | String | Tablespace name identifier. | `default`; `sysmaster` | Required          |
+
+### Metric: `db.disk.write.count`
+This metric is [optional](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#opt-in).
+
+| Name                    | Instrument Type  | Units (UCUM)  | Description                               |
+|-------------------------|------------------|---------------|-------------------------------------------|
+| `db.disk.write.count`   | UpDownCounter    | {write}       | Actual number of physical writes to disk. |
+
+### Metric: `db.disk.read.count`
+This metric is [optional](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#opt-in).
+
+| Name                 | Instrument Type    | Units (UCUM) | Description                              |
+|----------------------|--------------------|--------------|------------------------------------------|
+| `db.disk.read.count` | UpDownCounter      | {read}       | Actual number of physical reads to disk. |
+
 ## Maintenance Metrics
 ### Metric: `db.backup.cycle`
 This metric is [recommended](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#recommended).
 
 | Name              | Instrument Type | Units (UCUM) | Description    |
 |-------------------|-----------------|--------------|----------------|
 | `db.backup.cycle` | UpDownCounter   | `s`          | Backup cycle.  |
+
+
+## Settings Metrics
+
+### Metric: `db.database.log.enabled`
+This metric is [optional](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#opt-in).
+
+| Name                       | Instrument Type | Units (UCUM) | Description                                                   |
+|----------------------------|-----------------|--------------|---------------------------------------------------------------|
+| `db.database.log.enabled`  | Gauge           | by           | Database logging is enabled or not. 1 (Active), 0 (Inactive). |
+
+| Attribute              | Type   | Description               | Example                | Requirement Level |
+|------------------------|--------|---------------------------|------------------------|-------------------|
+| `db_database_name`     | String | Database name identifier. | `prod_db`; `sysmaster` | Required          |
```

Comment:
I think it's entirely possible to remove the `db_` prefix.

Reply:
Updated as suggested.